modules/Bio/EnsEMBL/Compara/PipeConfig/ncRNAtrees_conf.pm

```diff
@@ -181,6 +242,16 @@ sub pipeline_wide_parameters {  # these parameter values are visible to all anal
         'do_cafe'           => $self->o('do_cafe'),
         'dbID_range_index'  => $self->o('dbID_range_index'),
         'clustering_mode'   => $self->o('clustering_mode'),
+        'threshold_levels'  => $self->o('threshold_levels'),
+        'range_label'       => $self->o('range_label'),
+
+        'dna_alns_complete' => $self->o('dna_alns_complete'), # manually change to 1 when all wgas have finished
+        'orth_wga_complete' => $self->o('orth_wga_complete'), # populated by the rib_fire_orth_wga when wga analyses finished
```

Comment:
I can't see where this is being populated. `rib_fire_orth_wga` is a dummy that only flows to `pair_species`?

Reply:
I actually think it should be populated by an analysis in the `Parts/OrthologyQM_Alignment` so that you don't have to do it twice (once in ptrees, once in nctrees)

Reply:
It actually is populated by `check_file_copy` in `Parts/OrthologtQMAlignment.pl`, I think I just wrote a wrong comment there. I will fix the comment :facepalm: